### PR TITLE
fix(inline-loading): shrink spinner radius

### LINKED ISSUE #3040: [Inline loading] Update checkmark icon and loader size
**Checkmark icon**

Current:

New:
-Replace current check with `16/checkmark--filled`. Fill: `interactive-01` and Check: `icon-03`
-8px between icon and text

### PR DESCRIPTION
**Pull request by asudoh**
Fixes #3040.

#### Changelog

**Changed**

- The stroke radius of the spinner of inline loading.

#### Testing / Reviewing

Testing should make sure our inline loading component is not broken.

### CODE REVIEW

File: packages/components/src/components/inline-loading/_inline-loading.scss [hunk 1]
```diff
@@ -29,6 +29,12 @@
     .#{$prefix}--loading__stroke {
       stroke-dashoffset: 99;
     }
+
+    .#{$prefix}--loading__background,
+    .#{$prefix}--loading__stroke {
+      // Ensures (r * 2 + (stroke width)) / (viewbox unit size) * (viewbox pixel size) === 14
+      r: 26.8125;
```
Comment (emyarod) [inline packages/components/src/components/inline-loading/_inline-loading.scss]: it looks like Firefox still does not properly support modifying `r` with CSS?

Firefox 69 and 71

image

Chrome

image

**Review by laurenmrice — CHANGES_REQUESTED** [review]
The 16px loader should be used in three different places:

Vanilla
- small loader under Loading. (its currently using the 20px)
- inline loading (currently seems to be getting extra padding than the success icon)
- file uploader (looks correct)

React
- small loader under Loading. (its currently using the 20px)
- inline loading (its currently using the 20px)
- file uploader (looks correct size, but it is jumping to the left. It should stay in place like the complete and edit states )
Oct-18-2019 12-09-51

**Comment by asudoh** [comment]
Thank you for reviewing @laurenmrice - Updated the right padding for file uploader as well as the size for small loading and inline loading. I tried to make the spinner the size the same as complete/invalid icon (to get the same padding as them), but please let me know if a larger size is desired.

**Review by emyarod — APPROVED** [review]
Firefox issue is resolved, looks good to me pending design approval

**Review by laurenmrice — CHANGES_REQUESTED** [review]
The only place where it still needs to be update is in Vanilla under File uploader in the upload states example. It still too big here. 
Screen Shot 2019-10-21 at 10 43 07 AM

**Comment by asudoh** [comment]
Thank you for catching that @laurenmrice - Fixed.

**Review by laurenmrice — APPROVED** [review]
looks good, thank you !👍

**Review by abbeyhrt — APPROVED** [review]
Looks good to me!

**Pull request merged**